libs/integration-interface/src/lib/integration-interface.module.ts

```diff
@@ -0,0 +1,7 @@
+import { NgModule } from '@angular/core'
+import { CommonModule } from '@angular/common'
+
+@NgModule({
+  imports: [CommonModule],
+})
+export class IntegrationInterfaceModule {}
```

Comment:
Not needed because we are not using angular here. Please also check the accelerator lib